tests/k6/tests/performancetest_data/create-transmission.js

```diff
@@ -62,4 +62,112 @@ export default function (relatedTransmissionId) {
         transmission.relatedTransmissionId = relatedTransmissionId;
     }
     return transmission;
+}
+
+export function transmissionToInsertSkd(relatedTransmissionId, orgNo) {
+  let transmission = {
+      "id": uuidv7(),
+      "createdAt": new Date().toISOString(),
+      "authorizationAttribute": "element1",
+      "extendedType": "string",
+      "type": "Information",
+      "sender": {
+          "actorType": "partyRepresentative",
+          "actorId": "urn:altinn:organization:identifier-no:" + orgNo,
+      },
```

Comment:
Information skal være type serviceOwner, det er pt. ikke enforced, men kommer muligens en breaking change etter hvert 👀

```suggestion
      "sender": {
          "actorType": "serviceOwner"
      },
```